bingo.js

```diff
@@ -494,6 +495,14 @@ function changeDifficulty(value)
 	pushNewUrl();
 }
 
+function changeSeed(value)
+{
+	SEED = value;
+	//updateDifficulty();
```

Comment:
Noticed another nitpick: please don't commit commented out code, it will only confuse people who read the code later.